src/content/docs/alerts-applied-intelligence/new-relic-alerts/troubleshooting/tag-information-not-showing-entity-infra-alert-condition.mdx

```diff
@@ -12,7 +12,7 @@ redirects:
 
 ## Problem
 
-\[Custom violation description tags]([https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions#example](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions#example)) are not being replaced by the values from the event.
+[Custom violation description tags](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions#example) are not being replaced by the values from the event.
```

Comment:
```suggestion
[Custom violation description tags](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions#example) are not being replaced by the values from the event.
```

[nit] You can omit the site URL and use a relative path here. We do this under the hood, but I like to encourage it where possible.